src/node_worker.h

```diff
@@ -84,7 +84,9 @@ class Worker : public AsyncWrap {
   mutable Mutex mutex_;
 
   bool thread_joined_ = true;
-  const char* custom_error_ = nullptr;
+  std::string custom_error_;
```

Comment:
I think this one can stay the way it is now, i.e.

```suggestion
  const char* custom_error_ = nullptr;
```

Reply:
@addaleax, thanks. Fixed it.